src/System.Windows.Forms.Design/src/System/Windows/Forms/Design/TrackBarDesigner.cs

```diff
@@ -0,0 +1,50 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+
+namespace System.Windows.Forms.Design;
+
+internal class TrackBarDesigner : ControlDesigner
+{
+    public TrackBarDesigner()
+    {
+        AutoResizeHandles = true;
+    }
+
+    public override SelectionRules SelectionRules
+    {
+        get
+        {
+            SelectionRules rules = base.SelectionRules;
+            rules |= SelectionRules.AllSizeable;
+
+            if (GetPropertyValue<bool>(Component, nameof(TrackBar.AutoSize)))
+            {
+                var orientation = GetPropertyValue<Orientation?>(Component, nameof(TrackBar.Orientation));
+                if (orientation != null)
```

Comment:
This looks odd. Shouldn't it be
```suggestion
                if (orientation is null)
```
?